openeogeotrellis/backend.py

```diff
@@ -1153,6 +1156,98 @@ def request_costs(
 
 
 class GpsProcessing(ConcreteProcessing):
+
+    @staticmethod
+    def derive_temporal_extent(collection_id: str, constraints: dict, env: EvalEnv) -> List[Optional[str]]:
```

Comment:
Why is this under GpsProcessing? Isn't this more catalog aspect, so it better fits under GeoPySparkLayerCatalog , or even  a toplevel function at openeogeotrellis.layercatalog?

Reply:
Fits nice there. Moved!